src/Guide/Views.hs

```diff
@@ -793,3 +798,81 @@ on those <div>s.
 -- people instead just write “TODO fix grammar” in description and then such
 -- things could be displayed in gray font and also there'd be an
 -- automatically updated list of TODOs somewhere?)
+
+-- | Render links page with info about broken links
+renderAdminLinks :: (MonadIO m) => GlobalState -> HtmlT m ()
+renderAdminLinks globalState = do
+  head_ $ do
+    includeJS "/js.js"
+    includeJS "/jquery.js"
+    includeJS "/sorttable.js"
+    includeCSS "/markup.css"
+    includeCSS "/admin.css"
+    includeCSS "/loader.css"
+    title_ "Links – Aelve Guide"
+    meta_ [name_ "viewport",
+           content_ "width=device-width, initial-scale=1.0, user-scalable=yes"]
+
+  body_ $ do
+    script_ $ fromJS $ JS.createAjaxIndicator ()
+    h1_ "Links"
```

Comment:
Oops, I just noticed that non-broken links go into the table as well. Nevermind then :)